tests/Unit/NumericalAlgorithms/Spectral/Test_Spectral.cpp

```diff
@@ -517,7 +507,7 @@ void test_boundary_lagrange_polynomials() noexcept {
 SPECTRE_TEST_CASE("Unit.Numerical.Spectral",
                   "[NumericalAlgorithms][Spectral][Unit]") {
   test_streaming();
-  test_exact_diff_matrices();
+  test_exact_differentiation_matrices();
```

Comment:
This change should be in the other commit (I'm fine with just squashing this commit into the one that updates the tests if that's easier)